src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -59,7 +59,15 @@ public sealed class PSRunspaceDebug
         /// <param name="runspaceId">Runspace local Id.</param>
         public PSRunspaceDebug(bool enabled, bool breakAll, string runspaceName, int runspaceId)
         {
-            if (string.IsNullOrEmpty(runspaceName)) { throw new PSArgumentNullException(nameof(runspaceName)); }
+            if (runspaceName is null)
+            {
+                throw new PSArgumentNullException(nameof(runspaceName));
+            }
+
+            if (runspaceName.Length == 0)
+            {
+                throw new PSArgumentException(nameof(runspaceName));
```

Comment:
PSArgumentException doesn’t have a constructor that only takes a parameter name.

Reply:
Fixed in 42fb50d